ooniapi/services/ooniprobe/tests/conftest.py

```diff
@@ -69,20 +71,55 @@ def client_with_bad_settings():
 
 
 JWT_ENCRYPTION_KEY = "super_secure"
+GEOIP_DB_DIR = f"{tempdir}/ooni/geoip"
```

Comment:
You should define this path as a fixture and either make use of tmp_path of pytest (https://docs.pytest.org/en/stable/how-to/tmp_path.html), so that the tempfile is cleaned up after every test is run, or maybe even better cache it similarly to how this is done in pipeline (see: DATA_DIR in https://github.com/ooni/data/blob/main/oonipipeline/tests/conftest.py).

Reply:
I notice in pipeline that the data is stored in a file within the project directory, but I don't see how it's being cleared afterwards. Shouldn't be that function decorated with a `@pytest.fixture(scope='session')` with a clean up code after the yield? Or is not clearing it  the intended usage to improve iteration speed?

Reply:
Yeah the idea is to not clear it so that if you run tests multiple times you don't have to redownload the assets. This also allows to cache that path in github actions for faster test runtime.